Makefile

```diff
@@ -62,4 +62,12 @@ tag:
 package:
 	php build/packager.php
 
+static: static-phpstan static-cs-fixer
+
+static-phpstan:
+	docker run --rm -it -e REQUIRE_DEV=true -v ${PWD}:/app -w /app oskarstark/phpstan-ga:0.12.23 analyze
+
+static-cs-fixer:
+	docker run --rm -it -v ${PWD}:/app -w /app oskarstark/php-cs-fixer-ga:2.16.3.1 --dry-run --diff-format udiff
```

Comment:
maybe there should be a non-dry-run variant of this too. I know someone can pipe the output to git, but still, the point of a Makefile is that you have a simple command which does the most common use case, which would not be a dry run.

Reply:
Did remove the `--dry-run` parameter, as error code output also will be non-0 when changes were made.